doc/article/en-US/templating-custom-attributes.md

```diff
@@ -25,9 +25,8 @@ Many custom elements will work directly on the DOM element they are attached to.
 
 The simplest custom attribute to create is one that changes the behavior of an element merely by being added to the element. The following attribute will make an HTML element be displayed as a 100 pixel square with a red background.
 
-<code-listing heading="red-square.${context.language.fileExtension}">
+<code-listing heading="red-square${context.language.fileExtension}">
   <source-code lang="ES 2015/2016">
-
```

Comment:
removing empty lines breaks Markdown preview. I'd rather leave them here if they don't break anything in docs app.